### ISSUE TITLE
Dosn't work anymore

### ISSUE THREAD

**Issue by Aniloid2** [JeromeDane/YouTube-Remove-From-Feed-Button#16]
The program dons't work anymore following the recent youtube front end update

**Comment by JeromeDane** [comment]
This is true. Sadly I don't have the time/motivation to fix this right now, but anyone is welcome to fork this or create a pull request if they want.

**Comment by Montygue** [comment]
I dream of this extension or another working again someday. Was such a nice feature
